diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -224,15 +225,16 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
 
     @Test
     @Tag(WarningNames.WRONG_NEWLINES)
-    fun `chained calls should follow functional style - should not trigger on single dot calls`() {
+    fun `chained calls should follow functional style - should not trigger on single dot calls but not with prefix`() {
         lintMethod(
                 """
                     |fun foo(bar: Bar?) {
                     |    bar.baz()
                     |    bar?.baz()
                     |    bar!!.baz()
                     |}
-                """.trimMargin()
+                """.trimMargin(),
+                LintError(4,10,ruleId, "$functionalStyleWarn .", true)
```

Comment:
I think we only check cases with more than two calls?

Reply:
Yes, but as we [discussed](https://github.com/cqfn/diKTat/issues/492#issuecomment-732816549) `!!`  - is equivalent to dot - need line break